.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,12 +6,12 @@
 
 ## Mandatory Checklist
 
-- Please choose the target release of Azure PowerShell
+- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readyness**. Please click into the links for details.)
```

Comment:
- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click into the links for details.)